WordPress/WordPressTest/SiteCreation/SiteCreationIntentTracksEventTests.swift

```diff
@@ -40,13 +39,12 @@ class SiteCreationIntentTracksEventTests: XCTestCase {
     func testSiteIntentTracksEventFiresForTreatmentGroup() throws {
 
         // Given
-        let mockABTestForTreatment = SiteCreationIntentTests.SiteIntentABMock(variant: treatmentVariant)
-        let mockSiteCreator = SiteCreator()
+        let mockABTestForTreatment = SiteIntentAB.Variant.treatment
         let expectedEvent = WPAnalyticsEvent.enhancedSiteCreationIntentQuestionExperiment.value
         let expectedProperty = treatmentVariant.tracksProperty
 
         // When
-        let _ = SiteIntentStep(siteIntentAB: mockABTestForTreatment, creator: mockSiteCreator)
+        SiteCreationWizardLauncher.fireInitEvents(siteIntentVariant: mockABTestForTreatment, siteNameVariant: nil)
```

Comment:
I felt this event should live outside the step (and it doesn't even have to be coupled with the step's tests).